word_guess.rb

```diff
@@ -0,0 +1,325 @@
+                                                                     |  O O|
+                                                                    /___|_|/\_
+                                                           =======(|          |
+                                                                    (o)====(o)
+SEVEN
+@car6 = <<"SIX"
+                                                                       sweet syntactic sugar!
+                                                                _____ /
+                                                               |  O O|
+                                                              /___|_|/\_
+                                                       =====(|          |
+                                                              (o)====(o)
+SIX
+@car5 = <<"FIVE"
+                                                             infinite loop!
+                                                      _____ /
+                                                     |  O O|
+                                                    /___|_|/\_
+                                           =======(|          |
+                                                    (o)====(o)
+FIVE
+@car4 = <<"FOUR"
+                                                         oh fudge!
+                                                  _____ /
+                                                 |  O O|
+                                                /___|_|/\_
+                                         =====(|          |
+                                                (o)====(o)
+FOUR
+@car3 = <<"THREE"
+                                                     oh my stars!
+                                              _____ /
+                                             |  O O|
+                                            /___|_|/\_
+                                   =======(|          |
+                                            (o)====(o)
+THREE
+@car2 = <<"TWO"
+                                             goodbye cruel world!
+                                      _____ /
+                                     |  O O|
+                                    /___|_|/\_
+                              ====(|          |
+                                    (o)====(o)
+TWO
+@car1 = <<"ONE"
+                                       poop!
+                                _____ /
+                               |  O O|
+                              /___|_|/\_
+                         ===(|         |
+                              (o)====(o)
+ONE
+@dino_wins= <<'YAY'
+         __
+        /oo\
+       |    |
+   ^^  (vvvv)   ^^
+   \\  /\__/\  //
+    \\/      \//
+     /        \
+    |          |    ^
+    /          \___/ |
+   (            )    |
+    \----------/     /
+       //    \\_____/
+      W       W
+YAY
+
+
+intro = "Let's play a word guessing game!!!!"
+
+puts intro
+@play = GamePlay.new
```

Comment:
Instance variables are most often useful within classes. Since these variables are not within a class, I think it might make more sense to use local variables (variables without the `@`)